llvm-libunwind-16.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: llvm-libunwind-16
+  version: 16.0.6
+  epoch: 1
+  description: LLVM version of libunwind library
+  copyright:
+    - license: MIT
+  dependencies:
+    provides:
+      - llvm-libunwind=16
```

Comment:
Just for my own edification, how come this doesn't have something like 16.999

Reply:
I don't really know, I copied another llvm one :)